contrib/format-spss/src/test/java/org/apache/drill/exec/store/spss/TestSpssReader.java

```diff
@@ -108,7 +108,7 @@ public void testStarQuery() throws Exception {
 
   @Test
   public void testExplicitQuery() throws Exception {
-    String sql = "SELECT ID, Urban, Urban_value FROM dfs.`spss/testdata.sav` WHERE d16=4";
+    String sql = "SELECT ID, Urban, Urban_value FROM dfs.`spss/testdata.sav` WHERE d16=4 LIMIT 5";
```

Comment:
Why not write another test method with `LIMIT` instead of editing the existing one without a `LIMIT`?
In case with `LIMIT`, should query plan be validated also?

Reply:
@KazydubB 
I'll add additional unit tests this weekend.

Reply:
Fixed.